### ISSUE TITLE
feat(loss): Add MSE and Cross-Entropy loss functions

### ISSUE THREAD

**Issue by davidselorm** [davidselorm/py-autograd-engine#2]
Implement scalar loss computation layers for regression.
